### DISCUSSION TITLE
Public monitoring

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#833] [General]
Are we able to include a stat page on the public side through an addon maybe that would allow your public visitors to see how much usage the bandwidth is using at the moment?

**Comment by rejetto** [comment]
you mean the same total bandwidth you see in the admin-panel?

**Comment by Stoufs** [comment]
Correct. How much upload and download the bandwidth is currently using.

**Comment by rejetto** [comment]
i'm sure it's possible to make some code to show that information, but it's not super trivial, because it's currently not public (not just "hidden").
It takes a programmer to do the job, but in case you are, i can give you a hint, to use this line to access the information, in the backend.
```js
const { totalInSpeed, totalOutSpeed } = api.require('./throttler')
```
and then expose it, possibly with the new "customRest", or even better with "notifications" (all documented in dev-plugins.md)

I fail to see why this should be made public, TBH.
